### ISSUE TITLE
isFamilyFriendly 

### ISSUE THREAD

**Issue by BeatGrowth** [schemaorg/schemaorg#3293]
What exactly defines Family Friendly? Can't find the definition. 

I'm assuming it means things link strong language, blood/gore, nudity, etc.. but i don't know for certain.

**Comment by danbri** [comment]
Yeah, it is culturally varying. Roughly it is suitable for children, but
that is notoriously impossible to define. If sites have a field
approximating this, it gives applications a chance to use it with
discretion, but we won’t try to impose anything globally via the schemas
